### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by gerib**

### CODE REVIEW

File: reposilite-site/data/guides/features/dashboard.md [hunk 1]
```diff
@@ -3,36 +3,33 @@ id: dashboard
...
-You can also filter those messages using filters, but keep in mind it searches only though the currently cached log entries, not the whole history.
+You can also filter those messages using filters, but keep in mind that it searches only through the currently cached log entries, not the whole history.
 
 #### Shared configuration
 Shared configuration has been already mentioned in:
 
 * [Guide / Setup :: Shared configuration](/guide/settings#shared-configuration)
 
-Currently, configuration is provided in CDN format:
+Currently, configuration is provided in [CDN (_change detection and Notification_)](https://en.wikipedia.org/wiki/Change_detection_and_notification) format:
```
Comment (joshuasing) [inline reposilite-site/data/guides/features/dashboard.md:30]: I think `CDN format` is referring to [dzikoysk/cdn](https://github.com/dzikoysk/cdn) (`https://github.com/dzikoysk/cdn`), if so, would it be better to link to that instead?

File: reposilite-site/data/guides/features/dashboard.md [hunk 2]
```diff
@@ -3,36 +3,33 @@ id: dashboard
...
 
 #### Browser
-Browser displays content of currently selected directory, respectively to provided authentication credentials. For not logged user it lists only public files, for authenticated one it also displays files that user has access to.
+Browser displays the content of the currently selected directory in respect to the provided authentication credentials. For users not logged in it lists only public files, for authenticated users it also displays files that the user has access to.
 
 ![Dashboard Browser Preview](/images/guides/dashboard-browser-preview.png)
 
-You can control the way Reposilite displays directory content by opening adjustments view through the icon in right top corner:
+You can control the way how Reposilite displays directory content by opening the adjustments view via the icon in the top-right corner:
```
Comment (joshuasing) [inline reposilite-site/data/guides/features/dashboard.md:13]:
```suggestion
You can control how Reposilite displays directory content by opening the adjustments view via the icon in the top-right corner:
```

I'm not sure `the way how` is the best way of wording this, maybe just `how` would work better?

File: reposilite-site/data/guides/features/dashboard.md [hunk 3]
```diff
@@ -3,36 +3,33 @@ id: dashboard
...
-You can disable frontend in [shared configuration](/guide/settings#shared-configuration)
-if you just want to disable an easy way to query your repository or to provide custom frontend implementation using [static files](/guide/static-files).
+Reposilite exposes a frontend that is mounted to `/#/` as [SPA (_single-page application_)](https://en.wikipedia.org/wiki/Single-page_application) in [Vue 3](https://vuejs.org/). You can disable the frontend in [shared configuration](/guide/settings#shared-configuration) if you want to disable an easy way to query your repository or to provide a custom frontend implementation using [static files](/guide/static-files).
 
 #### Browser
-Browser displays content of currently selected directory, respectively to provided authentication credentials. For not logged user it lists only public files, for authenticated one it also displays files that user has access to.
+Browser displays the content of the currently selected directory in respect to the provided authentication credentials. For users not logged in it lists only public files, for authenticated users it also displays files that the user has access to.
```
Comment (joshuasing) [inline reposilite-site/data/guides/features/dashboard.md:9]:
```suggestion
Browser displays the contents of the currently selected directory in respect to the provided authentication credentials. For unauthenticated users, only public files will be listed. For authenticated users, all files that the user has access to will displayed.
```

I think these changes would help make this sentence clearer and more consistent.

**Review by joshuasing — COMMENTED** [review]
Hi, I had some spare time and had a look through your changes, I noticed a few things that could possibly be improved.

I hope my comments are helpful, thank you!

File: reposilite-site/data/guides/features/dashboard.md (hunk 2 above)
Comment (gerib) [reply reposilite-site/data/guides/features/dashboard.md:13]: You're right: https://ell.stackexchange.com/q/246368/80364. I let me confuse myself.

File: reposilite-site/data/guides/features/dashboard.md (hunk 1 above)
Comment (gerib) [reply reposilite-site/data/guides/features/dashboard.md:30]: You're right. Wasn't aware of that @dzikoysk developed his own CDN.

File: reposilite-site/data/guides/features/dashboard.md (hunk 3 above)
Comment (gerib) [reply reposilite-site/data/guides/features/dashboard.md:9]: With your suggestion there's a change of time in the paragraph: "Browser displays" – "will be listed" – "will [be] displayed". 

Change of "not logged in" to "unauthenticated" is a good idea.

**Review by dzikoysk — APPROVED** [review]

**Comment by dzikoysk** [comment]
Thanks, I had to miss the notification when I was on my holiday trip. Thanks for PR @gerib & @joshuasing for review! :)

**Pull request merged**